bridge/client/app/_components/ktb-project-settings-shipyard/ktb-project-settings-shipyard.component.html

```diff
@@ -1,4 +1,6 @@
-<h2>Shipyard <ng-container *ngIf="isCreateMode">*</ng-container></h2>
+<h2>Shipyard
+  <ng-container *ngIf="isCreateMode">*</ng-container>
+</h2>
```

Comment:
We have a 'required' class, that adds a red * at the end.

```suggestion
<h2 [class.required]="isCreateMode">Shipyard</h2>
```

Reply:
Actually this should then be adjusted across the complete `project settings` page, since this class was not used here. And also the info at the bottom should then use the `required-info` class.

Not sure if we should adjust this with this issue/PR?

Reply:
It is not part of the issue and the change here was just because of automated formatting. But I'll adjust it.